march_gazebo_plugins/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 2.8FATAL_ERROR)
```

CMake documentation says you need to add a space inbetween the version and `FATAL_ERROR`: https://cmake.org/cmake/help/v3.0/command/cmake_minimum_required.html
```suggestion
cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
```